### PR TITLE
Add kutercan's file

### PR DESCRIPTION
**Pull request by kutercan**
a new file guys

### CODE REVIEW

File: _posts/0000-01-02-kutercan.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (kutercan) [inline _posts/0000-01-02-kutercan.md:5]: it must be changed

Comment (kutercan) [reply _posts/0000-01-02-kutercan.md:5]: okay okay go on

**Comment by kutercan** [comment]
go mate go

**Review by kutercan — COMMENTED** [review]
okay. sounds good

**Review by kutercan — COMMENTED** [review]
okay

**Review by kutercan — COMMENTED** [review]
really bored

**Review by serdarkaygilar — APPROVED** [review]
great dude

**Review by serdarkaygilar — APPROVED** [review]
okay go on please

**Pull request merged**